src/git/squash_merge.go

```diff
@@ -1,8 +1,26 @@
 package git
 
-import "github.com/Originate/git-town/src/util"
+import (
+	"io/ioutil"
+	"strings"
 
-// CommentOutDefaultSquashCommitMessage comments out the default message for the current squash merge
-func CommentOutDefaultSquashCommitMessage() {
-	util.GetCommandOutput("sed", "-i", "-e", "s/^/# /g", ".git/SQUASH_MSG")
+	"github.com/Originate/git-town/src/exit"
+)
+
+var squashMessageFile = ".git/SQUASH_MSG"
+
+// CommentOutSquashCommitMessage comments out the message for the current squash merge
+// Adds the given prefix with the newline if provided
+func CommentOutSquashCommitMessage(prefix string) {
+	contentBytes, err := ioutil.ReadFile(squashMessageFile)
+	exit.On(err)
+	content := string(contentBytes)
+	if prefix != "" {
+		content = prefix + "\n" + content
+	}
+	lines := strings.Split(content, "\n")
```

Comment:
This can be simplified using regexp:

```go
content := regexp.MustCompile("(?m)^").ReplaceAllString(content, "# ")
```

See https://play.golang.org/p/hjSiDtxw8l for a working example.

I also suggest extracting this logic into a `commentOutCommitMessageText` function that can be unit tested separately.

Reply:
done